cfn-resources/federated-settings-org-role-mapping/Makefile

```diff
@@ -0,0 +1,11 @@
+.PHONY: build test clean
+
+build:
+	make -f makebuild  # this runs build steps required by the cfn cli
+
+test:
```

Comment:
This file is not matching with make file of  other modules .
please compare it with cluster module

Reply:
Fixed